{"md_content":"# Genetic diversity in captive Yellow Cardinals (Gubernatrix cristata) from Southern Brazil: implications for the management and conservation of an endangered species  \n\nSandra Eloisa Bülau1 $\\cdot$ Willian Thomaz Peçanha2 · Claiton Martins‑Ferreira $\\bullet$ Thales Renato Ochotorena de Freitas1,2  \n\nReceived: 7 July 2020 / Revised: 23 December 2020 / Accepted: 6 January 2021   \n$\\circledcirc$ Deutsche Ornithologen-Gesellschaft e.V. 2021  \n\n# Abstract  \n\nYellow Cardinal is an Endangered Species. Its populations are in constant decline due to habitat reduction and loss and illegal capture. There are approximately 50 free-living animals in the south of Brazil (state of Rio Grande do Sul), but estimates indicate more than 1000 in captivity. Our goals were to investigate the genetic variability of captive animals in Brazil, compare it with the genetic variability found in free-living populations, estimate the kinship of captive individuals, and obtain updated data about the number of captive animals. We found only $13\\%$ of the registered captive Yellow Cardinals. Analysis of mitochondrial sequences of captive and free-living populations was performed. We obtained 14 haplotypes, one of them (H14) exclusive of captive individuals. Furthermore, we found a haplotype exclusive to the La Pampa population in a male born in captivity in 2013. The most frequent haplotype in captive animals was the same reported for northern Argentina and Uruguay populations, which border Brazil. We found a high number of alleles by locus (mean $=9.6$ ) for microsatellite markers, with high average heterozygosity $(\\mathrm{HO}=0.605\\$ and $_{\\mathrm{HE}=0.727}$ ). The global F was 0.168. Kinship analysis of captive birds indicates a degree of kinship means of 0.082. In addition, the probability of relationship indicates that $79.4\\%$ of them are unrelated. Our data demonstrated that the captive group retains a high degree of genetic diversity with a low level of kinship. Nevertheless, the presence of haplotype exclusive for Argentina’s population in captive birds, and the low level of parentage among captive animals may be representing cases of illegal wildlife trade.  \n\nKeywords  Passeriformes $\\cdot$ Birds $\\cdot$ South america $^{\\ast}$ habitat reduction $\\cdot$ mtDNA $^{\\ast}$ Microsatellite  \n\n# Zusammenfassung  \n\n# Genetische Diversität bei in Gefangenschaft gehaltenen Grüntangaren Gubernatrix cristata aus Südbrasilien: Konsequenzen für Management und Schutz einer stark gefährdeten Vogelart  \n\nDie Grüntangare ist eine stark gefährdete Vogelart, deren Populationen aufgrund von schrumpfenden Lebensräumen und Habitatverlust sowie illegalem Fang stetig abnehmen. In freier Wildbahn leben etwa 50 Vögel im Süden Brasiliens (Bundesstaat Rio Grande do Sul), Schätzungen zufolge allerdings mehr als 1.000 in Gefangenschaft. Unser Ziel war es, die genetische Variabilität der Volierenvögel in Brasilien zu untersuchen und diese mit der genetischen Variabilität der freilebenden Populationen zu vergleichen sowie den Verwandtschaftsgrad der Individuen in Volierenhaltung zu bestimmen und aktualisierte Daten zur Anzahl der Vögel in Gefangenschaft zu erhalten. Wir konnten nur $13\\%$ der registrierten Grüntangaren in Gefangenschaft ausfindig machen. Wir führten Analysen mitochondrialer Sequenzen der gefangenen und freilebenden Populationen durch. Wir entdeckten 14 Haplotypen, von denen einer (H14) ausschließlich bei Individuen aus Gefangenschaft vorkam. Des Weiteren konnten wir einen ausschließlich in der La Pampa-Population vertretenen Haplotyp bei einem 2013 in Gefangenschaft geschlüpften Männchen nachweisen. Der bei den Volierenvögeln häufigste Haplotyp war derjenige, welcher von den Populationen im Norden Argentiniens und Uruguays an der Grenze zu Brasilien bekannt ist. Wir stellten eine hohe Anzahl von Allelen pro Genlokus (Mittelwert $=9{,}6.$ ) bei den Mikrosatellitenmarkern fest, bei einer hohen Durchschnittsheterozygosität $(\\mathrm{HO}=0,605\\$ und $\\mathrm{HE}=0\\mathrm{,}727.$ ). Der globale F-Wert betrug 0,168. Verwandtschaftsanalysen an Vögeln in Gefangenschaft deuten auf einen mittleren Verwandtschaftsgrad von $0{,}082~\\mathrm{hin}$ . Außerdem weist die Verwandtschaftswahrscheinlichkeit darauf hin, dass $79.4\\%$ der Individuen nicht verwandt sind. Unsere Daten zeigen, dass sich die Volierenpopulation ein hohes Maß an genetischer Diversität in Verbindung mit einem niedrigen Verwandtschaftsgrad bewahrt hat. Dennoch könnten das Vorhandensein eines für die argentinische Population typischen Haplotyps sowie der geringe Abstammungsgrad bei den Volierenvögeln Fälle illegalen Vogelhandels widerspiegeln.  \n\n# Introduction  \n\nTogether with wildlife trade, fragmentation and habitat loss are the main threats for bird species (IUCN 2019; Scheffers et al. 2019). Wildlife trafficking is among the most widespread and profitable illegal activity, and it has a direct and indirect negative impact on global biodiversity (Alacs et al. 2010; Lawson and Vines 2014; Alexander and Sanderson 2017; Kurland and Pires 2017). Birds are the richest class in animal species recorded in the world wildlife trade, and the most common orders are the Psittaciformes, Passeriformes, and Falconiformes (Bush et al. 2014). Birds trade is an intense illegal activity in Brazil, and the most captured orders follow the global pattern (Destro et al. 2012). This activity is indicated as a cause of extinction of Spix’s Macaw (Cyanopsitta spixii) and Hyacinth Macaw (Anodorhynchus hyacinthinus) and affects populations of numerous other species (Guix et al. 1997; Wright et al. 2001; Marini and Garcia 2005).  \n\nThreatened species require conservation practices based on genetic data (Presti and Wasko 2014). Genetic diversity assessment through molecular techniques allows inferences on population structure, gene flow level, dispersion patterns, recruitment, sexual selection, and kinship determination (Westneat and Webster 1994; De Woody 2005; Faria et al. 2008; Jones et al. 2010; Caparroz et al. 2011; Batalha-Filho et al. 2012; Cabanne et al. 2013; Biondo et al. 2014; Presti and Wasko 2014). In some cases, molecular forensic methods may determine the geographical origin of seized individuals and provide evidence of illegal trade (Ferreira et al. 2014; Gonçalves et al. 2015; Presti et al. 2015; Coetzer et al. 2017). In addition, knowledge of genetic diversity is necessary for developing management strategies.  \n\nGenetic management strategies applied to endangered species aim to preserve the genetic variability, minimize inbreeding, and inbreeding depression (Lacy 1993; Frankham et al. 2002; IUCN 2002). Moreover, they are useful for developing the captive breeding program, reintroduction, and translocation in natural populations (Miyaki et al. 1997; Wakchaure and Ganguly 2016; DeMay et al. 2017). Such programs’ success depends on consistent sets of genetics data used to construct studbooks, the inference of genealogies when ancestry is unknown, and the identification of individuals genetically important for ex-situ reproduction (Russello and Amato 2004).  \n\nThe Yellow Cardinal (Gubernatrix cristata) is an endangered bird on a global level, according to the International Union for Conservation of Nature (IUCN) categories of conservation status. Its total free-living population is estimated at 1000–2000 individuals (BirdLife International 2019). The species is endemic of the Pampa biome, with restricted geographical distribution. It is strongly associated with the vegetation type Savanna Parque (Shrubland) and depends on this ecosystem for survival (Azpiroz 2003; Bencke 2009; Martins-Ferreira et al. 2010). Historically, its distribution in Brazil extended from the Serra do Sudeste, to the border with Uruguay in the state of the Rio Grande do Sul, and on the Espinal ecoregion of Uruguay and Argentina, from the north to the center of this country (Ridgely and Tudor 1989; Collar et al. 1992; Di Giacomo 2005). A severe and persistent population decline has been observed throughout its range, and the remaining populations are small and isolated (Fontana et al. 2003; Di Giacomo 2005). Recent records have been obtained in areas of difficult access or preserved (Domínguez et al. 2020).  \n\nThe Yellow Cardinal has a high genetic diversity (Martins-Ferreira et al. 2010). However, despite a specimens’ release in different natural distribution areas, Argentina and Uruguay populations are genetically structured and form four independent management units. This differentiation is attributed to isolation-by-distance, habitat discontinuities, and the Parana-Paraguay basin geographical barrier between the northern and southern Argentinean populations (Domínguez et al. 2017). The Pampa biome landscape fragmentation is responsible for the current reduction in their distribution and reflects the geographical structure of genetic diversity found for the species (Martins-Ferreira et al. 2013; Domínguez et al. 2017).  \n\nThe destruction and fragmentation of Yellow Cardinal’s habitat and the capture for cage bird trade, mainly males, are the leading causes for decreasing their natural populations (Chebez 1994; Pessino and Tittarelli 2006; Ortiz 2008; Martins-Ferreira et al. 2013). Also, the growing records of natural hybridization between G. cristata and Diuca diuca in Argentina (Bertonatti and Guerra 1997; Pessino 2006), and the parasitism of their nests by the Shiny Cowbird, Molothrus bonarienses, and by botfly larvae of the genus Philornis sp., lead to a significant reduction of the reproductive success of the Yellow Cardinal (Domínguez et al. 2015). All these threats require the adoption of protective and management measures to maintain this species in natural conditions (Serafini et al. 2013).  \n\nThe Brazilian government has been developing conservation strategies for threatened species by implementing national action plans. The national action plan for the conservation of endangered passerines in the Campos Sulinos and Espinilho (Plano de Ação Nacional para a Conservação dos Passeriformes ameaçados dos Campos Sulinos e Espinilho) proposes conservation measures for 22 species of passerine birds, including the Yellow Cardinal. Action 2.3 of this plan deals specifically with the captive reproduction of this species. It is motivated by the small free-living population in Brazil, estimated as 50 animals, resident in the Espinilho State Park in Rio Grande do Sul (Serafini et al. 2013; Beier et al. 2017). However, according to Serafini et al. (2013), there are over a thousand captive Yellow Cardinals in Brazil, mainly with amateur breeders registered in the Amateur Bird Breeding Activity Control System (Sistema de Controle da Atividade de Criação Amadora de Pássaros—SISPASS). Registration on SISPASS is mandatory, but data upload is declaratory, i.e., the breeder himself provides both personal and bird data (IBAMA 2011). This self-reporting nature does not prevent fraud and makes this data vulnerable and inaccurate (Alves et al. 2010, 2013; Mayrink 2016).  \n\nThus, knowing information about these populations is crucial for the management strategies of captive individuals. Our objectives in this study were (1) to investigate the genetic variability of captive Yellow Cardinals in the state of Rio Grande do Sul, (2) to compare the genetic diversity of captive versus free-living populations, available on the GenBank, using mitochondrial DNA markers, (3) to estimate the level of the kinship of captive birds, and (4) to assess the real number of Yellow Cardinals kept in captivity by amateur breeders in the state of Rio Grande do Sul.  \n\n# Methods  \n\n# Sample collection  \n\nWe obtained the number of amateur breeders of Yellow Cardinals, registered in the SISPASS for the state of Rio Grande do Sul, from the database maintained by the Brazilian Institute of Environment and Renewable Natural Resources (Instituto Brasileiro do Meio Ambiente e dos Recursos Naturais Renováveis—IBAMA). In cooperation with the National Research Center for the Conservation of Wild Birds (Centro Nacional de Pesquisa e Conservação de Aves Silvestres— CEMAVE), we invited these breeders to participate in our research. From December 2013 to August 2014, we visited those who accepted our invitation.  \n\nWe collected blood in the cutaneous ulnar vein, using a sterile syringe, on filter paper, or FTA Cards (Whatman). When available, we removed three to four molting feathers, which were stored in Eppendorf tubes at $4~^{\\circ}\\mathrm{C}$ . We adopted the handling and sampling collection recommendations established by IBAMA (IBAMA 1994). In addition to the biological material, we recorded information regarding the date of birth, sex, and ring number (identification number) of each animal.  \n\n# Ethics statement, permissions, and storage  \n\nWe performed the blood and feather sampling with a license from the Brazilian environmental authorities (IBAMA— SISBIO 40,312–2). We deposited the biological material and DNA extracted in the tissue bank of the Cytogenetics and Evolution Laboratory of the Universidade Federal do Rio Grande do Sul. The breeders’ registration data and identification number of captive Yellow Cardinal were omitted in this study for confidentiality reasons.  \n\n# Laboratory procedures, alignment and genotyping  \n\nWe extracted the total DNA from the blood and feathers using the CTAB method (Doyle and Doyle 1987). We amplified a segment of the mitochondrial DNA (mtDNA) control region using the primers LCR3 and H1248 (Tarr 1995), chosen to have data compatibility with those already published. For the Polymerase Chain Reactions (PCR), we followed the protocol used by Dominguez et al. (2017). We stained the PCR products with GelRed (Biotium), verified in $1\\%$ agarose gel, purified by the enzymatic method (Exonuclease and Phosphatase alkaline; Amersham Biosciences, Piscataway, NJ). The PCR products were sequenced by Macrogen (Seoul, Korea), in an automatic sequencer (ABI 3730XL) with Capillary Electrophoresis Sequencing method, using both primers (forward and reverse). We deposited all sequences obtained in GenBank under the accession codes: MW029496 – MW029606. We visualized and edited the mtDNA sequences in the Chromas Lite program $\\phantom{+}\\mathrm{v}2.6.5$ (https://technelysium.com.au/wp/) and used MEGA X program v10.0.3 (Kumar et al. 2018) with the Muscle algorithm to automatically align them.  \n\nTo assess the nuclear genetic variability, we used ten pairs of species-specific primers (GcrisC02; GcrisC08; GcrisE02; GcrisG10; GcrisF02; GcrisF12; GcrisH06; GcrisH07; GcrisH09 and GcrisH12) developed by Martins-Ferreira et al. (2010). Moreover, we followed the protocol established by the authors for amplifying microsatellites. We genotyped the PCR products in an automatic sequencer (ABI 3730XL) at René Rachou Research Center. We repeated $10\\%$ of the genotyping of each locus for every 100 genotyped samples. We failed in amplifying the GcrisG10 locus. We determined the allele size using Peak Scanner v1.0 (http://www.appli edbiosystems.com/absite/us/en/home.html).  \n\n# mtDNA genetic diversity and demographic history  \n\nWe analyzed the mtDNA genetic diversity using samples obtained from captive birds and sequences taken from GenBank, which correspond to five wild populations (Uruguay; and Corrientes, San Luis, La Pampa, and Río Negro from Argentina). We adopted the nomenclature and \"geographic arrangement\" of each population’s haplotypes in territories of Argentina and Uruguay, according to the results from Dominguez et al. (2017).  \n\nLevels of genetic diversity were determined estimating parameters such as the number of segregating sites (S), number of haplotypes (NH), haplotype diversity (h), nucleotide diversity $(\\pi)$ , and average number of differences between pairs (k) using the programs DNASP v5.10. (Librado and Rozas 2009) and Arlequin v3.5. (Excoffier and Lischer 2010). We also obtained the number of private haplotypes $\\mathrm{(Ph)}$ (i.e., haplotypes that occur in only one region) and the haplotypes with the highest incidence in each population.  \n\nStatistic tests Tajima’s $D$ (Tajima 1989), and Fu’s $F s$ (Fu 1997) were performed to identify deviations from neutral models in DNASP v5.10 (Librado and Rozas 2009). The neutrality tests were employed to reveal the historical changes, such as demographic expansion signatures and equilibrium or decline. These changes were examined for each population, considering captive sampling.  \n\nThe pairwise mismatch distribution was obtained using DNASP v5.10 (Librado and Rozas 2009), calculated with the expected frequency based on the growth-decline model to infer the Yellow Cardinal’s historical demographics. In addition to the resulting plot, the sum of the square deviations (SSDs) (Schneider and Excoffier 1999) between the observed and expected mismatch distribution, and the  \n\nHarpending’s raggedness index (r) (Harpending 1994) were calculated to test the null hypothesis of spatial expansion using Arlequin v3.5 (Excoffier and Lischer 2010). Significance of r and SSD were assessed through 1000 bootstrap replicates. For this analysis, three approaches were considered: only wild populations, only captive group, and the total dataset (wild $+$ captive group). This latter approach considers both captive and wild populations included as from the original natural stock.  \n\nTo explore genetic differentiation via mtDNA, we calculated the paired FST among all populations (including captivity sampling) pairs using 10,000 random permutations using Arlequin v3.5 (Excoffier and Lischer 2010).  \n\n# Microsatellite loci diversity and kinship of captive birds  \n\nWe assessed the genetic diversity for microsatellite data considering the total number of alleles per locus, observed and expected heterozygosity, and Hardy–Weinberg equilibrium for each locus in Arlequin v3.5 (Excoffier and Lischer 2010). We corrected the significance level values of the Hardy–Weinberg equilibrium by correcting Bonferroni $\\alpha{=}0.005,$ . We evaluated the discriminatory power of microsatellite loci by calculating the values of the polymorphic information content (PIC) per locus in the Cervus v3.0 (Kalinowski et al. 2007). Loci with PIC values higher than 0.5 are considered very informative, values between 0.25 and 0.50 are moderately informative, and with values lower than 0.25 poorly informative (Botstein et al. 1980). We calculated the global inbreeding coefficient F using the formula $\\mathrm{F}{=}1{-}(\\mathrm{Ho/He})$ (Frankham et al. 2002).  \n\nTo estimate the degree of kinship $(r)$ and the probabilities of relationship (R) between individuals, we used the MLRelate program (Kalinowski et al. 2006). This program considers the population’s allelic frequency to suggest the kinship among pairs of individuals using a maximum likelihood approach. The degree of kinship is given by an absolute scale (0–1). Ml-Relate also allows the identification of relationship probabilities in unrelated (UR), full-sib (FS), half-sib (HB), and parent-offspring (PO) categories by identifying the highest probability [ML(R)] (Kalinowski et al. 2006; Wagner et al. 2007). We tested all loci for null alleles with the test for heterozygote deficiency relative to Hardy–Weinberg’s expectations with ML-Relate. Null alleles were detected in four loci (GcrisC02; GcrisF12; GcrisH06; GcrisH12). Their frequencies were estimated with ML-Relate using a maximum likelihood method (Kalinowski and Taper 2006). We applied the correction for the presence of null alleles, this correction increases the accuracy of kinship and relationship probabilities. This approach also eliminated the probability of exclusion of relatives and calculated the probability of false exclusion of a candidate relative for the loci in which null alleles were detected, multiplying the observed heterozygosity by the corrected allele frequency $(\\mathrm{HO^{*}P n})$ , where Pn is the frequency of null alleles for the locus (Wagner et al. 2006).  \n\n# Results  \n\nThe initial dataset provided by the IBAMA indicated the presence of the 1,223 individuals of Yellow Cardinals, kept by 432 amateur breeders from 21 localities in the Rio Grande do Sul. After reviewing data, only 71 amateur breeders were located, and 64 agreed to participate in our research. Thus, considering the records, 194 amateur breeders were not found mainly due to inaccurate or outdated location data. Another 164 breeders did not report data on Yellow Cardinals specimens, and ten chose not to participate (Appendix 1). We found 172 Yellow Cardinals and sampled 168 (100 males and 68 females), representing only $13\\%$ of the registered number of animals.  \n\n# Control region diversity in Yellow Cardinals  \n\nWe obtained the mtDNA control region with 738 bp of 205 individuals (111 from captive birds’ samples and 94 sequences from GenBank). The detailed description of the sample, both for the data produced in this study and from GenBank, is presented in Table S1. The entire data set included 12 variable sites (724 sites conserved), five singletons, and seven informative parsimony sites. We did not detect indels in sequence alignments. Considering all sampling (wild $^+$ captive group), we observed 14 haplotypes, 13 of which were present in wild populations. All wild populations, except for Río Negro, had private haplotypes. Regarding the captive group, a single private haplotype was observed (H14). The distribution and frequency of haplotypes found in the species’ geographic distribution can be verified in Fig. 1 and Table 1.  \n\nThe most frequent haplotypes also varied according to the data group analyzed. Considering wild populations, the haplotypes: H5 $(41.48\\%)$ , H1 $(36.17\\%)$ , and H6 $(7.44\\%)$ were the most frequent, while for the whole data set, H1 $(58.53\\%)$ , H5 $(19.51\\%)$ , and H2 $(7.31\\%)$ were the most frequent. For the captive group, the haplotype H1 had $77.47\\%$ frequency, while its private haplotype (H14) was observed in a single individual with a frequency lower than $1\\%$ $(0.90\\%)$ . We found the H5 and H2 haplotypes 1 and 11 times, respectively, H2 being the second most frequently found in this study. Other haplotypes occurred with low frequencies (Table 1 and Fig. 1).  \n\nThe H12 haplotype found in only one individual of the Uruguayan population was present in six animals sampled in captivity. The same occurs with the H4 haplotype, also considered exclusive to the Uruguayan population found in four of the animals sampled in this study, and with H8, considered exclusive to the La Pampa population that we found in a captive male with birth registered as 2013.  \n\nDiversity indices varied among the wild and the captive group (Table 2). The total $h$ was 0.6957 for the wild group, 0.3889 for the captive one, and 0.6133 for the whole database. For wild populations, the $h$ ranges from 0.3333 (San Luis) to 0.6166 (Uruguay). Nevertheless, the $\\pi$ and k ranged from 0.00045 (San Luis) to 0.00127 (Uruguay) and from 0.33333 (San Luis) to 0.93676 (Uruguay), respectively. In general, the wild populations had the highest genetic diversity level, while the captive birds had the lowest one.  \n\nFor wild populations, the diversity indices for the population Uruguay presented the highest values in general, while the populations San Luis and Corrientes showed the lowest ones. The captive group of Rio Grande do Sul presented lower values than those observed for wild populations in general, except when comparing the populations San Luis and Corrientes (Table 2).  \n\n# Population demographic history and genetic differentiation  \n\nFor wild populations, the Tajima’s $D$ and Fu’s $F s$ neutrality tests (Table 2) were negative and not significant for the San Luis and La Pampa populations. Both tests were significant $(P<0.05)$ for population Corrientes. Simultaneously, only Fu’s $F s$ was significant $(P<0.01)$ for population Uruguay. Río Negro population was the only one showing positive and non-significant values, which could indicate of population contraction (few segregation sites/many differences between pairs for Tajima’s $D$ ) and/or recent bottleneck effect (positive values of Fu’s $F s$ ). However, considering all the wild animals as a single group, it exhibited negative and significant values for both parameters, indicating that Yellow Cardinal populations may have experienced recent population expansion. As for the captive group and the whole sample (wild $+$ captive group), although negative for both the neutrality tests, only Fu’s $F s$ was found significant $(P<0.01)$ .  \n\nThe results of the mismatch distribution for mtDNA control region analysis were approximately unimodal (Fig. 2). However, considering only data from wild populations (Fig. 2a), the SSD and raggedness index value statistics were significant (SSD: 0.01411384, $P=0.0055$ ; r: 0.12800921, $P=0.002,$ . The significant SSD value obtained suggests evidence of deviation in the estimated demographic model of a sudden population expansion. In addition, the high and significant raggedness index suggests that this population experienced stationary patterns or bottleneck effect. In contrast, the results of the captive group (SSD: 0.00054094, $P=0.7940$ ; r: 0.15535701, $P=0.7210$ ) and total dataset (SSD: 0.00390422, $P{=}0.0980$ ; r: 0.08917493, $P{=}0.0190;$  \n\n![](images/f069472d8908a8c5a676bbb31fb3f270106017d1600bae46ea039ceeda8b76c5.jpg)  \nFig. 1   Map of geographic range of Yellow Cardinal’s (Gubernatrix cristata). The thick lines represent the patterns of extant/resident (orange), possibly extant (purple) and possibly extinct (red checkered) in South America, according to the IUCN Red List of Threatened Species (BirdLife International, 2019). Locations of wild populations: Uruguay (U; $33^{\\circ}53^{\\prime}54^{\\prime\\prime}\\mathrm{S}$ , $54^{\\circ}44^{\\prime}33^{\\prime\\prime}\\mathrm{W})$ , Corrientes (C1; $29^{\\circ}12^{\\prime}6^{\\prime\\prime}5$ , $58^{\\circ}15^{\\prime}41^{\\prime\\prime}\\mathrm{W}$ and C2; $28^{\\circ}39^{\\prime}22^{\\prime\\prime}\\mathrm{S}$ , $57^{\\circ}26^{\\prime}4^{\\prime\\prime}\\mathrm{W},$ , San Luis (SL; $32^{\\circ}25^{\\prime}10^{\\prime\\prime}\\mathrm{S}$ , $66^{\\circ}53^{\\prime}27^{\\prime\\prime}\\mathrm{W},$ ), La Pampa (LP1; $36^{\\circ}12^{\\prime}15^{\\prime\\prime}\\mathrm{S}$ , $65^{\\circ}05^{\\prime}58^{\\prime\\prime}\\mathrm{W}$ , LP2; $36^{\\circ}38^{\\prime}6^{\\prime\\prime}5$ , $64^{\\circ}17^{\\prime}48^{\\prime\\prime}\\mathrm{W}$ and LP3; $37^{\\circ}42^{\\prime}52^{\\prime\\prime}\\mathrm{S}$ , $64^{\\circ}46^{\\prime}14^{\\prime\\prime}\\mathrm{W}_{\\cdot}$ )   \nand Rio Negro (RN1; $40^{\\circ}3^{\\prime}9^{\\prime\\prime}\\mathrm{S}$ , $64^{\\circ}8^{\\prime}38^{\\prime\\prime}\\mathrm{W}$ and RN2; $40^{\\circ}24^{\\prime}28^{\\prime\\prime}\\mathrm{S}$ , $63^{\\circ}40^{\\prime}54^{\\prime\\prime}\\mathrm{W})$ following Dominguez et  al. (2017). The state of Rio Grande do Sul in Brazil, representing the captive group, is highlighted (Dashed lines). For mtDNA, the pie charts circles show each population’s haplotypes. Colors haplotypes: H1 dark blue, H2 purple, H3 pink, H4 red, H5 reddish orange, H6 orange, H7 yellow gold, H8 light yellow, H9 light green, H10 green, H11 cyan, H12 blue, H13 grey and H14 light grey  \n\nTable 1   Description of general estimates of most common haplotype, identification and number of private haplotypes $(P_{h})$ for different populations of Yellow Cardinals   \n\n![](images/0382f831403902cdceff13363c84462aea39ff0230d7a60f68c74ec9eff98dcc.jpg)  \nIn parenthesis indicates the percentage of haplotypes within populations  \n\nTable 2   Genetic diversity indices and neutrality tests for Yellow Cardinals using mitochondrial control region sequences $(738~\\mathrm{bp})$ for wild and captive populations   \n\n![](images/a9c9b03c5916177fa12d3071611e1bf11b211ef4dcfb98dc245909cc68b262f5.jpg)  \nThe indices are shown as $n$ number of samples, S Number of segregating sites, $N_{\\mathrm{H}}$ number of haplotypes, $h$ Haplotype diversity, $\\pi$ Nucleotide diversity, $k$ Mean number of pairwise differences, $S D$ Standard deviation. Fu’s $F s$ and Tajima’s $D$ tests were considered statistically significant when $^{*}P$ -value $<0.01$ and $^{**P}$ -value $<0.05$  \n\npresented SSD statistics and raggedness index value nonsignificant (considering $P>0.01\\$ ), thus indicating that the curves fit the sudden expansion model (Fig. 2b and c).  \n\nThe significant $F_{S T}$ values obtained from mtDNA analysis ranged from $-0.03493$ to 0.64950 (Table 3), considering only the wild populations. The pairwise $F_{S T}$ values between populations Uruguay and Corrientes were low (0.04308), indicating low genetic differentiation in the northern part of the geographic distribution. The other populations towards the south present the lowest genetic differentiation and nonsignificant $F_{S T}$ values among them (San Luis, Río Negro, and La Pampa). High and significant differences were only observed when we analyzed the two populations from the north (Uruguay and Corrientes) compared to those from the south (San Luis, Río Negro, and La Pampa) of the geographic distribution. The captive group presents similar patterns along with the north-to-south range with low genetic differentiation compared to populations that border Brazil (Uruguay, $F_{S T};$ : 0.04774; Corrientes, $F_{S T}{\\mathrm{:}}$ : -0.01704) and high and significant differences when compared to southern Argentina’s population. (San Luis, $F_{S T}\\colon0.6186$ ; Río Negro, $F_{S T}\\colon0.61281$ ; La Pampa, $F_{S T}\\colon0.57920\\rangle$ .  \n\n# Microsatellites diversity and parentage in captive Yellow Cardinals  \n\nWe genotyped 168 Yellow Cardinals, for eight microsatellite loci and 166, for nine loci. We present the values of the diversity measures in Table 4. The number of alleles per locus varied between 5 and 15 alleles, with an average of 9.6 alleles per locus. Observed heterozygosity (HO) ranged from 0.280 to 0.899, the mean being 0.605 $(\\pm0.202)$ , and expected heterozygosity (HE) ranged from 0.617 to 0.878, with the mean 0.727 $(\\pm0.082)$ . The Hardy–Weinberg equilibrium test indicated that five loci are out of equilibrium. The global F value was calculated as 0.168, and the PIC values obtained ranged from 0.55 to 0.86, with a mean of 0.69.  \n\nKinship analysis for the 168 captive Yellow Cardinals resulted in 14,028 pairs. The degree of kinship $(r)$ between the pairs ranged from 0.0 to 0.83, with a mean of 0.082. For male-male pairs, the $r$ ranged from 0.0 to 0.73 with a mean of 0.083. For female-female pairs, $r$ ranged from 0.0 to 0.76 with a mean of 0.081. Regarding the probability of kinship (MLR), $79.4\\%$ of pairs were classified as UR; $15.7\\%$ HS; $2.7\\%$ FS and $2.2\\%$ with PO. For male-male pairs, we obtained $79.1\\%$ probabilities as UR; $16.3\\%$ HS; $2.5\\%$ FS and $2.2\\%$ PO. Female-female pairs were estimated as $80.2\\%$ UR; $15.2\\%$ HS; $2.4\\%$ FS and $2.2\\%$ PO.  \n\n# Discussion  \n\nReproduction in captivity is not always the best alternative to avoid removing animals from nature, especially for endangered species, since there will always be a demand for new breeders (Coetzer et al. 2017). Thus, captive breeding may represent an additional risk factor for threatened species. In many cases, the number of animals with registered births in captivity is not plausible (White et al. 2012; Lyons and Natusch 2014). Animals are found without rings, with broken or violated rings, and even with broken legs, indicating that the breeding activity actually serves as a way to legalize animals taken from nature (Camargo et al. 2010; Alves et al. 2013; Mayrink 2016).  \n\nIllegal capture for trading is one of the main threats to the conservation of the Yellow Cardinal. There are reports and records of specimens captured in the wild to be kept in captivity throughout the species’ distribution (Collar et al. 1992; Fontana et al. 2003; Pessino and Tittarelli 2006; MartinsFerreira et al. 2013; Serafini et al. 2013). Yellow Cardinals are apprehended annually in Brazil, either in amateur breeders, irregularities found, or in operations to combat traffic and trade in open markets throughout the country (Camargo et al. 2010; Alves et al. 2013; Souza et al. 2014).  \n\n![](images/d8cc70c3b41e577950038775857d459b8897d9c06f56e2dc18fc6bbac8988e6d.jpg)  \nFig. 2   Demographic history of Yellow Cardinals with signatures for population expansion for the control region (mtDNA). a mismatch distributions of pairwise differences of wild populations haplotypes obtained under a model allowing expansion; $\\mathbf{b}$ mismatch distributions of pairwise differences of the captive group; c Overall dataset (wild $^+$ captive group)  \n\nOur study found only $13\\%$ of the Yellow Cardinals registered in the SISPASS. Many of the breeders included in the IBAMA’s dataset were not found at the registered address, or the address did not exist at all. In some cases, we found the breeders, but we did not find their registered birds. This is related to the fact that the registration in SISPASS is selfdeclaratory, not prone to fraudulent activities (Alves et al. 2010), ending up with inaccurate records (Destro et al. 2012; Alves et al. 2013). Moreover, there is a lack of monitoring and controlling procedures of the breeders’ activity. Inspection is needed mainly due to the relation between this activity and the illegal trade of wild birds (Destro et al. 2012; Alves et al. 2013).  \n\nTable 3 $F$ -Statistics based on pairwise estimates of control region (mtDNA) haplotype frequencies of Yellow Cardinals from five wild populations and captive group/Rio Grande do Sul in South America   \n\n![](images/feb39f4c5713c37b00cb8c97c491679db684e3da2f17d19c63b6c5c048d15ac5.jpg)  \nSignificant tests were performed using 10,000 permutations. Bold values are statistically significant Values in bold are significant: $P<0.01$  \n\nThe analysis of the captive group using mtDNA markers showed genetic variation lower than that found for their neighbouring wild population (Uruguay) and similar values to population Corrientes. Compared with other wild populations from the south (i.e., Argentinean populations), the indices are not discrepant. Our results show that the captive group has low genetic diversity, but it is not different from the other localities treated as a single group, even presenting the largest number of haplotypes compared to wild populations (although less than the total number of haplotypes of the wild specimens). The $F_{S T}$ values from mtDNA indicate a lack of structure in the captive group and suggest that this group is not entirely isolated, especially in locations close to both the north and the south of its distribution.  \n\nAccording to Dominguez et al. (2017), the H5 haplotype was the most frequent in the southern region of Chaco (San Luís) and the Espinal ecoregion (center and south) in Argentina, while H1 was more frequent in individuals sampled in Uruguayan population and northern Argentina (Corrientes). The captive group, analyzed here, also had H1 haplotype as the most frequent. We have no information regarding the genetic composition of the free-living population in Espinilho State Park, Brazil. Therefore, the frequency of haplotypes obtained for the captive group shows a similarity between this group in Brazil and populations in northern Argentina and Uruguay’s western populations.  \n\nAn intriguing fact is the presence of haplotypes, in the captive group, that are exclusive to some populations in Argentina and Uruguay. The Yellow Cardinals present a genetic structure in its distribution that allows tracing the geographical origin of the animals (Domínguez et al. 2017). Thus, finding captive Yellow Cardinals carrying exclusive haplotypes to Argentinean populations is an indication that these animals were captured in those areas and brought to Brazil. These patterns reinforce the idea that probably the maintenance of genetic diversity in the captive group of Yellow Cardinals is a consequence of the wild population diversity that existed in the Rio Grande do Sul state (natural stock) or even the illegal trade between nearby regions since the entry of new individuals can only occur through exchanges between breeders.  \n\nTable 4   Genetic parameters at the nine microsatellite loci in captive Yellow Cardinals samples from Southern Brazil. Number of genotyped animals, Alleles per locus, frequency of observed (Ho) and   \nexpected (He) heterozygosity, polymorphic information content (PIC) and estimated frequencies of null alleles $(P_{\\mathrm{n}})$   \n\n![](images/45eaa7eaecf06762b06424455753dc79e738f64b55f426256ba717c22a25961b.jpg)  \n\\*Loci that showed statistically significant deviation from Hardy–Weinberg proportions are given  \n\nRegarding the whole sample of wild birds, the neutrality tests Tajima’s $D$ (Tajima 1989) and Fu’s $F s$ (Fu 1997) resulted in negative and significant values, which are consistent with population expansion. Such a pattern is surprising, mainly because of the long history of habitat loss and capture resulted in only about $\\sim2000$ free-living animals. However, despite the mismatch distribution being unimodal, suggesting that a priori the populations may have undergone recent expansion, the results showed incongruity between the SSD and raggedness index significant values, indicating a poor fit to the stepwise growth model. Still, it is noteworthy that mismatch distribution tests are very conservative (Ramos-Onsins and Rozas 2002) and should be analyzed carefully. Thus, stationary patterns or bottleneck effect may have occurred with free-living animals.  \n\nAs for the captive group, although significant values of Tajima’s $D$ were observed only for the wild populations, the results of Fu’s $F_{S}$ test, which is based on the distribution of haplotypes, showed negative, significant values, indicating an excess of rare haplotypes over what would be expected under neutrality (as would be expected from a recent population expansion). This characteristic is consistent with data from the captive group, since the H1 haplotype is concentrated in almost $77.5\\%$ of such group, and the other eight haplotypes found in captive animals are distributed in just over $20\\%$ of the population.  \n\nMismatch distribution for the captive group has shown a unimodal pattern. Moreover, the non-significant values for SSD and $r$ in the goodness-of-fit test for this group suggest that population expansion occurred recently (Rogers 1995). It is worth mentioning that the captive group has individuals exchanged between amateur breeders. Before the National Action Plan for the conservation of endangered passerines in the Campos Sulinos and Espinilho began, some breeders received animals from apprehensions carried out by Brazilian authorities (C. MartinsFerreira, pers. comm.). However, as the migration rate increases between demes, population range expansion can produce the same molecular signature as population demographic expansion (Excoffier 2004). In this sense, the population increase is limited to the breeding facilities’ ability to insert new animals. The detection of population expansion reinforces that trade still occurs in Brazil and within neighboring regions. Moreover, the results of non-structuring $F_{S T}$ values for the northern populations, already discussed above, lead to the same conclusion.  \n\nThe whole sample (captive $+$ wild animals) shows high haplotype diversity and low nucleotide diversity. Such haplotype diversity is a pattern typically present in expanding populations (Avise 2000). Thus, our data may be interpreted as a signature of rapid demographic expansion from a smallsized effective population (Avise 2000). Due to negative and significant values from Fu’s $F s$ test and mismatch distribution consistent with an excess of low-frequency haplotypes, the total stock assessed herein is characteristic from recent population expansion, genetic hitchhiking, or from selective sweep (Fu 1997; Ramos-Onsins and Rozas 2002). The signatures of historical demography evidenced by mtDNA support the data obtained by the nuclear markers.  \n\nIn our study, microsatellite markers indicate that the captive group shows genetic diversity similar to that found in natural populations of Yellow Cardinals (Martins-Ferreira et al. 2010; Domínguez et al. 2017). We identified a high number of alleles per locus and high heterozygosity levels compared with other threatened bird species (Kvist and Rytkönen 2006; Chan et al. 2011; Presti and Wasko 2014; Presti et  al. 2015; Coetzer et  al. 2017). Lawrence et  al. (2008), when studying the levels of genetic diversity in a critically endangered sea bird species, the Magenta Petrel (Pterodroma magentae), also found high diversity in mitochondrial and nuclear genomes as other closely related birds with varying degrees of threat.  \n\nThe samples collected for this study are supposed to come from animals reproduced and bred in captivity. Besides, they belong to different generations, since we collected samples from individuals born from 1994 to 2014. Moreover, amateur breeders usually carry out specific mating to maintain or obtain a specific plumage color or singing tone. These factors break the basis of Hardy–Weinberg’s equilibrium principles (Frankham et al. 2002) and may be responsible for the deviations found in some loci.  \n\nThe PIC values indicate that the microsatellites used are informative for discriminating the genetic polymorphism of loci. All of them presented values above 0.50 and are useful in kinship analysis (Botstein et al. 1980). However, in our initial analyses, we identified null alleles in some loci, and the corrections were performed to minimize the risk of exclusion of true relatives in kinship analysis (Wagner et al. 2006, 2007).  \n\nKinship analyses indicate that individuals in the captive group are poorly related because $79\\%$ of the pairs tested were unrelated. Although we have a larger number of males sampled, we found no differences in the kinship values obtained when comparing male-male and femalefemale. Furthermore, the low $r$ values found indicate that the animals evaluated are not related, or may carry new alleles (Kalinowski et al. 2006; Konovalov and $\\mathrm{Heg}~2008)$ . Many animals with low $r$ are consistent with the low F value (0.168) obtained in this study. This value is similar to the total found by Dominguez et al. (2017) $\\mathrm{F}{=}0.141$ ), although in the populations of Corrientes $\\mathrm{F}{=}0.266,$ ) and Rio Negro $\\mathrm{F}{=}0.219$ ), the values are moderate, as well as that found by Martins-Ferreira et al. (2010) $\\mathrm{\\Delta}\\mathrm{F}{=}0.221$ ).  \n\nA positive result from our study is the data showing that the captive Yellow Cardinals maintain the genetic diversity observed in nature. Such data also help understand the source of the large genetic variability and low kinship level among the captive birds. However, the finding of captive individuals carrying exclusive alleles from distant regions, the large percentage of unrelated animals, and the low inbreeding coefficient suggest the existence of illegal international trade and point to the need for monitoring and surveillance of the amateur breeding activity, or even the cessation of reproduction by amateur breeders. The Yellow Cardinal’s remaining populations are structured in four independent management units (Dominguez et al. 2017), and there is evidence of release in the wild of captive birds born (Dominguez et al. 2017; 2020). Also, captive breed reproduction without genetic monitoring, as in amateur breeding, can negatively impact species genetic diversity and have serious implications for its conservation (Lacy 1993; Miyaki et al. 1997; Frankham et al. 2002; IUCN 2002; Russello and Amato 2004). All these factors should be considered when planning specie’s breeding and reintroduction programs. Finally, we recognize the urge to study the only free-living population in Brazil, with both mitochondrial and microsatellite markers, especially of the individuals in the Brazilian captive breeding program, is essential for the conservation of this endangered species.  \n\nSupplementary Information  The online version contains supplementary material available at https://doi.0rg/10.1007/s10336-021-01857-1.  \n\nAcknowledgements  This research was supported by the Coordenação de Aperfeiçoamento de Pessoal de Nível Superior (CAPES) through fellowship for Sandra Eloisa Bülau and Willian Thomaz Peçanha, Fundação de Amparo à Pesquisa do Estado do Rio Grande do Sul (FAPERGS) and Conselho Nacional de Desenvolvimento Científico e Tecnológico (CNPq). Thales Renato Ochotorena de Freitas received financial support from CAPES, CNPq, and FAPERGS. The authors are grateful to Centro Nacional de Pesquisa e Conservação de Aves Silvestres (CEMAVE) and Instituto Brasileiro do Meio Ambiente e recursos Naturais Renováveis (IBAMA) for all their support and assistance with this research. We also thank the Programa de Desenvolvimento Tecnológico em Ferramentas para a Saúde – PDTISFIOCRUZ for the use of nucleotide sequencing facilities at Instituto René Rachou – Fiocruz Minas.  \n\nAuthor contributions  Sandra Eloisa Bülau and Thales Renato Ochotorena de Freitas conceptualized. Sandra Eloisa Bülau and Willian Thomaz Peçanha analyzed the data. Sandra Eloisa Bülau and Claiton Martins-Ferreira performed the fieldwork. Thales Renato Ochotorena de Freitas supervised the study. All the authors discussed the results and wrote the article.  \n\nFunding  This research was supported by the Coordenação de Aperfeiçoamento de Pessoal de Nível Superior (CAPES), Fundação de Amparo à Pesquisa do Estado do Rio Grande do Sul (FAPERGS) and Conselho Nacional de Desenvolvimento Científico e Tecnológico (CNPq).  \n\nAvailability of data and material (data transparency)  All DNA sequences from this study are available on GenBank, Accession Numbers MW029496 – MW029606.  \n\n# Compliance with ethical standards  \n\nConflicts of interest  The authors declare that they have no conflict of interest.  \n\nEthics approval (include appropriate approvals or waivers)  This work was carried out with the authorization of Brazilian Institute of Environment and Renewable Natural Resources (IBAMA)–Brazilian Biodiversity Authorization and Information System (SISBIO) under licenses $\\Nu^{\\ o}$ 40312–1; 40312–2. All information about the licenses required to carry out this work is present in the ‘Methods’ section.  \n\n# References  \n\nAlacs EA, Georges A, FitzSimmons NN, Robertson J (2010) DNA detective: A review of molecular approaches to wildlife forensics. Forensic Sci Med Pathol 6:180-194. https://doi.org/10.1007/ s12024-009-9131-7   \nAlexander KA, Sanderson CE (2017) Battling the illegal wildlife trade. Nat Ecol Evol 1:0090. https://doi.org/10.1038/s41559-017-0090 Journal of Ornithology Alves RRN, De Farias Lima JR, Araujo HFP (2013) The live bird trade in Brazil and its conservation implications: An overview. Bird Conserv Int 23:53-65. https://doi.org/10.1017/S095927091   \n200010X Alves RRN, Nogueira EEG, Araujo HFP, Brooks SE (2010) Birdkeeping in the Caatinga, NE Brazil. Hum Ecol 38:147–156. https​ ://doi.org/10.1007/s10745-009-9295-5 Avise JC (2000) Phylogeography: The History and Formation of Species. Harvard University Press, Harvad Azpiroz AB (2003) Aves Del Uruguay: Lista e introducción a su biología y conservación. Aves Uruguay-GUPECA, Montevideo Batalha-Filho H, Cabanne GS, Miyaki CY (2012) Phylogeography of an Atlantic forest passerine reveals demographic stability through the last glacial maximum. Mol Phylogenet Evol 65:892–902. https​ ://doi.org/10.1016j.ymev.2012.08.010 Beier C, Repenning M, Pereira MS et al (2017) Cooperative breeding and demography of yellow cardinal Gubernatrix cristata in Brazil. Revista Brasileira de Ornitologia 25:12-19. https://doi. org/10.1007/bf03544371 Bencke GA (2009) Diversidade e conservação da fauna dos Campos do Sul do Brasil. In: Pillar VDP, Müller SC, Castilhos ZM de S, Jacques AVA (eds) Campos Sulinos: conservação e uso sustentável da biodiversidade. MMA, Brasília, pp 101–121 Bertonatti C, Guerra AL (1997) Hibridación entre cardenal amarillo (Gubernatrix cristata) y diuca comun (Diuca diuca minor) en estado silvestre, en la Argentina. El Hornero 14:235–242 Biondo C, Izar P, Miyaki CY, Bussab VSR (2014) Social structure of collared peccaries (Pecari tajacu): Does relatedness matter? Behav Proc 109:7078. htps://doi.org/10.1016/j.beproc.2014.08.018 BirdLife International (2019) Species factsheet: Gubernatrix cristata. In: Species factsheet: Gubernatrix cristata. http://datazone.birdl ife.org/species/factsheet/22721578. Acessed 13 Mar 2019 Botstein D, White RL, Skolnick M, Davis RW (1980) Construction of a genetic linkage map in man using restriction fragment length polymorphisms. Am J Hum Genet 32:314–331 Bush ER, Baker SE, Macdonald DW (2014) Global trade in exotic pets   \n2006-2012.ConservBiol28:663-676.https://doi.org/10.1111/ cobi.12240​ Cabanne GS, Sari EHR, Meyer D et al (2013) Matrilineal evidence for demographic expansion, low diversity and lack of phylogeographic structure in the Atlantic forest endemic Greenish Schiffornis Schiffornis virescens (Aves: Tityridae). J Ornithol 154:371–   \n384. https://doi.org/10.1007/s10336-012-0901-8 Camargo CMJ, Camargo LJJ, Sueiro RF (2010) A criação amadora de pássaros nativos e os danos ambientais à fauna do Mato Grosso do Sul. Atualidades Ornitológicas 155 [on li:56–61 Caparroz R, Miyaki CY, Baker AJ (2011) Genetic evaluation of the mating system in the blue-and-yellow macaw (Ara ararauna, Aves, Psittacidae) by DNA fingerprinting. Genet Mol Biol   \n34:161–164 Chan CH, Robertson HA, Saul EK et al (2011) Genetic variation in the kakerori (Pomarea dimidiata), an endangered endemic bird successfully recovering in the Cook Islands. Conserv Genet 12:441–   \n447. https://doi.org/10.1007/s10592-010-0152-2 Chebez JC (1994) Los que se van: especies Argentinas en peligro. Albatros, Buenos Aires Coetzer WG, Downs CT, Perrin MR, Willows-Munro S (2017) Testing of microsatellite multiplexes for individual identification of Cape Parrots ( Poicephalus robustus ): paternity testing and monitoring trade. PeerJ 5:e2900. https://doi.org/10.7717/peerj.2900 Collar NJ, Gonzaga LP, Krabbe N, et al (1992) Threatened birds of the Americas: The I.C.B.P. / I.U.C.N. Red Data Book, Third. Smithsonian Inst. Press, ICBP, Cambridge De Woody JA (2005) Molecular approaches to the study of parentage, relatedness, and fitness: practical applications for wild animals. J Wildlife Manag 69:1400-1418. https://doi.org/10.2193/0022- 541x(2005)69[1400:mattso]2.0.co;2   \nDeMay SM, Becker PA, Rachlow JL, Waits LP (2017) Genetic monitoring of an endangered species recovery: Demographic and genetic trends for reintroduced pygmy rabbits (Brachylagus idahoensis). J Mammal 98:350-364. htps://doi.org/10.1093/jmamm al/gyw197   \nDestro GFG, Pimentel TL, Sabaini RM et al (2012) Efforts to combat wild animals trafficking in Brazil. In: Lameed GA (ed) Biodiversity Enrichment in a Diverse World. InTech, Rijeka, pp 33–36   \nDi Giacomo AS (ed) (2005) Áreas importantes para la conservación de las aves en Argentina: sítios prioritários para la conservación de la biodiversidad. Temas de Naturaleza y Conservácion, 5th edn. Aves Argentinas/Asociación Ornitológica del Plata, Buenos Aires   \nDomínguez M, Lapido R, Gorrindo A et al (2020) A citizen science survey discloses the current distribution of the endangered Yellow Cardinal Gubernatrix cristata in Argentina. Bird Conserv Int. https://doi.org/10.1017/S0959270920000155   \nDomínguez M, Reboreda JC, Mahler B (2015) Impact of shiny cowbird and botfly parasitism on the reproductive success of the globally endangered yellow cardinal Gubernatrix cristata. Bird Conserv Int 25:294-305.https://doi.org/10.1017/S095927091400015X   \nDomínguez M, Tiedemann R, Reboreda JC et al (2017) Genetic structure reveals management units for the yellow cardinal (Gubernatrix cristata), endangered by habitat loss and illegal trapping. Conserv Genet 18:1131-1140. https://doi.org/10.1007/s1059 2-017-0964-4   \nDoyle JJ, Doyle JL (1987) A rapid isolation procedure for small quanties of freash leaf tissue. Phitochem Bull 9(1):11–15   \nExcoffier L (2004) Patterns of DNA sequence diversity and genetic structure after a range expansion: lessons from the infinite-island model. Mol Ecol 13:853-864. https://doi.org/10.1046/j.1365- 294X.2003.02004.x   \nExcoffier L, Lischer HEL (2010) Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows.MolEcolResour 10:564567.https://doi.org/10.1 111/j.1755-0998.2010.02847.x   \nFaria PJ, Guedes NMR, Yamashita C et al (2008) Genetic variation and population structure of the endangered Hyacinth Macaw (Anodorhynchus hyacinthinus): Implications for conservation. Biodivers Conserv 17:765-779.https://doi.org/10.1007/s10531-007-9312-1   \nFerreira JM, Burnham-Curtis M, Schunck F et al (2014) Development of novel polymorphic microsatellite markers for four bird species exploited by the illegal wildlife trade in Brazil. Conserv Genet Resour 7:435-436. htps://doi.org/10.1007/s12686-014-0389-z   \nFontana CS, Bencke GA, Reis RE (2003) Livro vermelho da fauna ameaçada de extinção no Rio Grande do Sul. EDIPUCRS, Porto Alegre   \nFrankham R, Ballou JD, Briscoe DA (2002) Introduction to Conservation Genetics. Cambridge University Press, Cambridge   \nFu YX (1997) Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection. Genetics 147:915–925   \nGonçalves PFM, Oliveira-Marques AR, Matsumoto TE, Miyaki CY (2015) DNA barcoding identifies illegal parrot trade. J Hered 106:560-564. https://doi.org/10.1093/jhered/esv035   \nGuix JC, Jove L, Ruiz X (1997) Muestreos del comercio de psitácidos neotropicales en la ciudad de Barcelona, España: 1991–1996. Ararajuba 5:159–167   \nHarpending H (1994) Infertility and forager demography. Am J Phys Anthropol93:385-390.https://doi.org/10.1002/ajpa.1330930310   \nIBAMA (2011) Instrução Normativa $\\mathrm{N}^{\\mathrm{o}}10/2011$ , de 19 de setembro de 2011. Dispõe sobre as atividades dos criadores amadoristas e comerciais de passeriformes da fauna nativa brasileira. Diário Oficial da União, Brasília, Brasil   \nIBAMA (1994) Manual de anilhamento de aves silvestres, 2nd edn. Instituto Brasileiro do Meio Ambiente e dos Recursos Naturais Renováveis, Brasília   \nIUCN (2019) The IUCN red list of threatened species. Version 2018–2. https://www.iucnredlist.org/. Accessed 20 Jan 2019   \nIUCN (2002) IUCN technical guidelines on the management of exsitu populations for conservation. https://doi.org/10.1017/CB097 81107415324.004   \nJones AG, Small CM, Paczolt KA, Ratterman NL (2010) A practical guide to methods of parentage analysis. Mol Ecol Resour 10:6–30. https://doi.org/10.1111/j.1755-0998.2009.02778.x   \nKalinowski ST, Taper ML (2006) Maximum likelihood estimation of the frequency of null alleles at microsatellite loci. Conserv Genet 7:991-995. https://doi.org/10.1007/s10592-006-9134-9   \nKalinowski ST, Taper ML, Marshall TC (2007) Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment. Mol Ecol 16:1099–1006   \nKalinowski ST, Wagner AP, Taper ML (2006) ML-Relate: a computer program for maximum likelihood estimation of relatedness and relationship. Mol Ecol Notes 6:576-579. https://doi.org/10.111 1/j.1471-8286.2006.01256.x   \nKonovalov DA, Heg D (2008) A maximum likelihood relatedness estimator allowing for negative relatedness values. Mol Ecol Resour 8:256-263. https://doi.org/10.1111/j.1471-8286.2007.01940.x   \nKumar S, Stecher G, Li M et al (2018) MEGA X: Molecular evolutionary genetics analysis across computing platforms. Mol Biol Evol 35:1547-1549. https://doi.org/10.1093/molbev/msy096   \nKurland J, Pires SF (2017) Assessing U.S. wildlife trafficking patterns: how criminology and conservation science can guide strategies to reduce the illegal wildlife trade. Deviant Behavior 38:375–391. https://doi.org/10.1080/01639625.2016.1197009   \nKvist L, Rytkönen S (2006) Characterization of a secondary contact zone of the Great Tit Parus major and the Japanese Tit P. minor (Aves: Passeriformes) in Far Eastern Siberia with DNA markers. Zootaxa 73:55–73   \nLacy RC (1993) Impacts of inbreeding in natural and captive populations of vertebrates: implications for conservation. Perspect Biol Med 36:480-496.https://doi.org/10.1353/pbm.1993.0081   \nLawrence HA, Taylor GA, Millar CD, Lambert DM (2008) High mitochondrial and nuclear genetic diversity in one of the world’s most endangered seabirds, the Chatham Island Taiko. Conserv Genet 9:1293-1301. https://doi.org/10.1007/s10592-007-9471-3   \nLawson K, Vines A (2014) Global impacts of the illegal wildlife trade: the costs of crime, insecurity and institutional erosion. Chatham House, Londres   \nLibrado P, Rozas J (2009) DnaSP v5: a software for comprehensive analysis of DNA polymorphism data. Bioinformatics 25:1451– 1452.https://doi.org/10.1093/bioinformatics/btp187   \nLyons JA, Natusch DJD (2014) Wildlife laundering through breeding farms: illegal harvest, population declines and a means of regulating the trade of green pythons (Morelia viridis) from Indonesia. Biolog Conserv 144:073-3081. htps://doi.org/10.1016/j.bioco n.2011.10.002   \nMarini MA, Garcia FI (2005) Conservação de aves no Brasil. Megadiversidade 1:95–102   \nMartins-Ferreira C, Repenning M, Damiani RV (2013) Gubernatrix cristata. In: Serafini PP (ed) Plano de ação nacional para a conservação dos passeriformes ameaçados dos campos sulinos e espinilho. Instituto Chico Mendes de Conservação da Biodiversidade, ICMBio, Brasília, pp 116–119   \nMartins-Ferreira C, Santos MO, Hadddrath O et al (2010) Isolation and characterization of 10 microsatellite loci in the Yellow Cardinal Gubernatrix cristata En. Mol Ecol Resour 10:751–754   \nMayrink RR (2016) Exame pericial para detecção de fraudes em anilhas oficiais de passeriformes: Uma ferramenta para o combate ao tráfico de animais silvestres. Dissertation ,Programa de Pós-graduação em Perícias Criminais Ambientais, Universidade Federal de Santa Catarina, Florianópolis, Brasil. (In Portuguese)   \nMiyaki CY, Pereira SL, Biasia I, Wantjal A (1997) DNA fingerprinting applied to parrot captive breeding program. Ararajuba 5:127–133   \nOrtiz D (2008) Distribución histórica y actual del cardenal amarillo (Gubernatrix cristata) en el litoral fluvial argentino. In: Aceñolaza FG (ed) Temas de la biodiversidad del litoral fluvial argentino, III. São Miguel de Tucumán, pp 121–126   \nPessino M, Tittarelli RF (2006) Cardenal amarillo (Gubernatrix cristata): diagnóstico de su situación con respecto al comercio ilegal en la provincia de La Pampa, Argentina. Gestión Ambiental 12:69–76   \nPessino MEM (2006) Nuevo registro y descripción de híbridos entre cardenal amarillo (Gubernatrix cristata) y diuca común (Diuca diuca) en la provincia de La Pampa, Argentina. Revista Nuestras Aves 52:16–18   \nPresti FT, Guedes NMR, Antas PTZ, Miyaki CY (2015) Population genetic structure in hyacinth macaws (Anodorhynchus hyacinthinus) and identification of the probable origin of confiscated individuals. J Hered 106:491-502. https://doi.org/10.1093/jhere d/esv038   \nPresti FT, Wasko AP (2014) A review of microsatellite markers and their application on genetic diversity studies in parrots. Open J Genet 04:6977.htps://doi.org/10.4236/jgen.2014.42010   \nRamos-Onsins SE, Rozas J (2002) Statistical properties of new neutrality tests against population growth. Mol Biol Evol 19:2092–2100. https://doi.org/10.1093/oxfordjournals.molbev.a004034   \nRidgely RS, Tudor G (1989) The birds of South America, vol I. University of Texas Press, Austin, The Oscine Passerines   \nRogers AR (1995) Genetic Evidence for a Pleistocene Population Explosion. Evolution 49:608–615   \nRussello MA, Amato G (2004) Ex situ population management in the absence of pedigree information. Mol Ecol 13:2829–2840. https​ :/doi.org/10.1111/j.1365-294X.2004.02266.x   \nScheffers BR, Oliveira BF, Lamb I, Edwards DP (2019) Global wildlife trade across the tree of life. Science 366:71-76. https://doi. org/10.1126/science.aav5327   \nSchneider S, Excoffier L (1999) Estimation of past demographic parameters from the distribution of pairwise differences when the mutation rates vary among sites: application to human mitochondrial DNA. Genetics 152:1079–1089   \nSerafini PP, Martins-Ferreira C, Bencke GA et al (2013) Plano de ação nacional para a conservação dos passeriformes ameaçados dos campos sulinos e espinilho. Instituto Chico Mendes de Conservação da Biodiversidade, ICMBio, Brasília   \nde Souza T, O, Vilela DA da R, Câmara BG de O, (2014) Pressões sobre a avifauna brasileira: Aves recebidas pelo CETAS/IBAMA, Belo Horizonte, Minas Gerais. Ornithologia 7:1–11   \nTajima F (1989) The effect of change in population size on DNA polymorphism. Genetics 123:597–601   \nTarr CL (1995) Primers for amplification and determination of mitochondrialn control-region sequences in oscine passerines. Mol Ecol 4:527–529   \nWagner AP, Creel S, Frank LG, Kalinowski ST (2007) Patterns of relatedness and parentage in an asocial, polyandrous striped hyena population. Mol Ecol 16:4356-4369. https://doi.org/10.1111/ j.1365-294X.2007.03470.x   \nWagner AP, Creel S, Kalinowski ST (2006) Estimating relatedness and relationships using microsatellite loci with null alleles. Heredity 97:336-345. https://doi.org/10.1038/sj.hdy.6800865   \nWakchaure R, Ganguly S (2016) Captive Breeding in Endangered Wildlife: a Review. J Biolog Sci Opin 4:186-187. https://doi. org/10.7897/2321-6328.04544​   \nWestneat DF, Webster MS (1994) Molecular analysis of kinship in birds: Interesting questions and useful techniques. In: Schierwater B, Streit B, Wagner GP, DeSalle R (eds) Molecular Ecology and Evolution: Approaches and Applications. Experientia Supplementum, Birkhäuser, Basel, pp 91–126   \nWhite NE, Dawson R, Coghlan ML et al (2012) Application of STR markers in wildlife forensic casework involving Australian blackcockatoos (Calyptorhynchus spp.). Forensic Sci Int Genet 6:664– 67. https://doi.org/10.1016/j.fsigen.2011.10.003   \nWright TF, Toft CA, Enkerlin-Hoeflich E et al (2001) Nest poaching in neotropical parrots. Conserv Biol 15:710-720. https://doi.org/ 10.1046/j.1523-1739.2001.015003710.x  \n\nPublisher’s Note  Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.  "}