{"md_content":"# Original Article An innovative and cost-effective way to estimate alkaline phosphatase activity in in vitro cellular model systems  \n\nPoonam Kanta, Tulikalipi Ghosh, Anit Kaur, Thungapathra Muthukumarappa  \n\nDepartment of Biochemistry, Postgraduate Institute of Medical Education and Research, Chandigarh 160012, India  \n\nReceived June 22, 2020; Accepted November 24, 2020; Epub February 15, 2021; Published February 28, 2021  \n\nAbstract: Alkaline phosphatase is an enzyme that converts para-nitrophenyl phosphate to para-nitrophenol (yellow coloured) in 2-amino, 2-methyl, 1-propanol buffer at pH 10.5. However, when this protocol is applied to the in vitro cellular model systems to estimate alkaline phosphatase activity, it tends to generate clumps of genomic DNA, lead­ ing to inaccurate pipetting for protein estimation. The aim of the study was to introduce minor modifications in the existing protocol to make it simple, cost-effective, with minimal labor-intensive procedures while estimating alkaline phosphatase activity in cellular model systems. The genomic DNA clumps were dissolved by depurination (adding 0.2 N HCl) and fragmentation (adding 0.2 N NaOH) during enzyme estimation. Moreover, these minor modifications have been standardized and optimized extensively by using serum samples (rich source of alkaline phosphatase), hFOB/ER9 (human Fetal osteoblastic cell) and HepG2 cells. Our results suggest that the modification incorporated in previously published method was robust enough to estimate ALP activity and protein concentration accurately. There was no significant variation in ALP activity estimated after modification $\\mathrm{\\Delta}\\mathrm{P}>0.05$ ). This innovative approach could be beneficial for a researcher by providing an easy, cost effective and less labor-intensive solution for estima­ tion of enzymatic activity in cellular model systems.  \n\nKeywords: Alkaline phosphatase, para-nitrophenol, para-nitrophenyl phosphate, sonication  \n\n# Introduction  \n\nAlkaline phosphatase (ALP) assay, which includes the estimation of its activity in serum is a common biochemical test done for the assessment of various pathological conditions [1-4]. This assay measures the conversion of para-nitrophenyl phosphate to a yellow coloured compound, i.e. para-nitrophenol (PNP) at pH 10.5 [5]. The biochemical estimation of ALP activity is the same irrespective of the tis­ sue source (Liver, Kidney and Bone), which secretes this enzyme [6-8]. The ALP activity is estimated in serum by directly mixing with 2-amino, 2-methyl, 1-propanol (AMP) buffer at pH 10.5 followed by estimation of absorbance of the resultant yellow color solution at $405~\\mathrm{nm}$ [9]. However, when ALP activity has to be esti­ mated in the cells similar to the published pro­ tocol [10] grown in culture plates, addition of  \n\nAMP buffer containing substrate i.e. para-nitro­ phenyl phosphate (pNPP) results in lysis of the cells leading to the release of genomic DNA along with the cellular content. Presence of genomic DNA clumps contributes to absor­ bance and inaccurate protein estimation when ALP activity is estimated. Although various pro­ tocols have been adapted to estimate ALP activity in the cultured cells, the modified meth­ od of estimation presented here is simple and cost effective as it does not require any kit.  \n\nAs ALP activity is expressed in ${\\mathsf{U}}/{\\mathsf{m g}}$ of protein [11], there is a need to quantify the protein. However, the viscosity of the solution makes it difficult to quantitate the protein con­ tent accurately. Minor changes in protein con­ centration values can change the ALP activity. To avoid these artifacts of the protocol reported earlier [10, 12], new protocols were reported  \n\n# Detection of ALP in cellular model systems  \n\n[13] in which the cells were trypsinized, lysed in buffer by sonication to reduce the DNA viscosi­ ty. This sonicated extract was used to estimate the ALP activity and DNA content by Picogreen dye method. Finally, ALP activity of different cell cultures was normalized with respect to the total DNA content. The main disadvantage of above method was labor-intensive sonication procedures for individual cell cultures and use of an expensive Picogreen kit for estimation of total DNA content. Hence, there was a need to develop a method which allows the analysis of several parameters viz., analysis of gene expression, proteins and ALP activity all from a single set of treated cells.  \n\nIn this study we aim to develop more efficient, economical, fast and consistent method to estimate ALP activity and protein concentration in a single tube for cultured cells. Some minor modifications were incorporated in the existing protocol [10] and extensive optimizations were performed. It is well-known that when DNA is treated with dilute acid $(0.2N{\\cdot}0.4\\mathrm{~N~H~C~}$ l) fol­ lowed by alkali, the DNA gets depurinated and hydrolysis of phosphodiester bonds at the depurinated points results in breakage of DNA into small fragments [14-16]. The resulting solution with fragmented DNA is no more vis­ cous and thus it is possible to pipette out accu­ rate volumes. By this modification, the viscosity which always resulted in inaccurate protein concentration estimation was solved. We pre­ sumed that addition of acid followed by alkali treatment to the lysed cells should not cause any interference for both the ALP assay as well as the Bradford assay. The same treatments were performed for all the standards as well as all the samples to nullify the variation. Thus, ALP activity and protein concentration in cellu­ lar model systems was measured accurately by this method.  \n\n# Materials and methods  \n\n# Study design and study subjects  \n\nSample collection procedures were in accor­ dance with the ethical standards of the respon­ sible committee on human experimentation (Institutional Ethics committee no. NK/802/ MD/2121). After enrolment in the study from each subject, a written informed consent was obtained after explaining the protocol. The study was conducted in the Department of  \n\nBiochemistry. A total of 8 serum samples were used for the study.  \n\nCell lines and materials  \n\nHepatocellular carcinoma cell line HepG2 was purchased from NCCS Pune, India and hFOB/ ER9 cell line was obtained as gift from Dr M. Subramaniam, Mayo Clinic, Rochester, USA. Both cell lines were maintained in DMEM: F12 Ham’s media (Sigma-Aldrich, USA) with $10\\%$ FBS (GIBCO, Thermofisher, Scientific USA) and 1x antibiotic and antimycotic solution (PAA Laboratories, Yeovil, UK) at ${37}^{\\circ}{\\mathrm{C}}$ and $5\\%$ ${\\mathrm{CO}}_{2}$ . The cells were trypsinized using $0.05\\%$ Trypsin-EDTA solution (HiMedia Laboratories Ltd., India) and maintained at early passages. 2- amino, 2-methyl, 1-propanol (HiMedia Laboratories Ltd., India), Disodium para nitrophenyl phosphate (AR), p-nitrophenol (HiMedia Laboratories Ltd., India) were used in the study.  \n\n# ALP assay  \n\nApproximately $\\mathsf{1}\\times\\mathsf{10^{4}}$ cells per well were plated in 96 well plate. The cells were maintained for three days with a change of media. Alkaline phosphatase assay was performed after the cells attained confluence. To each well of 96 well plate $0.1~\\mathrm{mL}$ of AMP buffer pH 10.5 con­ taining $2\\mathrm{mg/mL}$ of para-nitrophenyl phosphate was added. After mixing, the cell lysate was incubated at ${37^{\\circ}}{\\mathrm{C}}$ for 15 minutes followed by addition of $0.006~\\mathsf{m L}\\textsf{6N H}$ Cl and mixed. This was immediately followed by addition of 0.104 $\\mathsf{m L1.0}\\mathsf{N}\\mathsf{N a0}\\mathsf{H}$ , which insures the termination of ALP activity as well as neutralization of HCl (Supplementary Information).  \n\n# Bradford assay  \n\nFor protein estimation, Bradford method was extensively optimized to estimate the strength of Bradford reagent in presence of AMP buffer. A $10~\\upmu\\updownarrow$ aliquot was taken from the tube after ALP measurement and to that desired volume of Bradford reagent was added. The volume of Bradford Reagent and total reaction volumes in standards and test samples should be the same. The volumes could be equalized using 0.15 N NaCl [17].  \n\nStatistical analysis was performed using Graph Pad Prism 5.0. T-test was performed to com­ pare two methods. One-way ANOVA was per­ formed to calculate the significance between different methods (acid treatment, sonication and conventional) for the estimation of ALP in serum samples. $P$ -value $<0.05$ was considered significant.  \n\n![](images/4d65849f325a4111987500b717bb6228b711be921b3072e2d346658deee07283.jpg)  \nFigure 1. Effect of acid treatment on ALP standards (PNP). Different concentrations of Para-nitrophenol (0.00054, 0.00108, 0.00216, 0.00324, 0.00432, $0.0054\\upmu\\mathrm{mol}.$ ) were used to compare between pres­ ence and absence of acid. Total volume was 0.320 mL.  \n\n# Results  \n\n# Effect of acid treatment on standards  \n\nAddition of HCl to a final concentration of $0.2\\mathrm{~N~}$ would have depurinated the DNA, which is fol­ lowed by addition of NaOH would break, the DNA strands. This step would have reduced the viscosity of the DNA in solution. However, addi­ tion of HCl followed by NaOH should not affect the accuracy of the assay. So, the effect of HCl on the standard (PNP) was studied experimen­ tally. To one set of a series of PNP standards only HCl was added followed by NaOH and in a second set, neutralized HCl was added as it does not have acidic property after neutraliza­ tion. We observed no significant difference in the value of standards as shown in the curve plotted after addition of acid followed by neu­ tralization $(\\mathsf{y}=0.0008\\mathsf{x}-0.0069$ , $\\mathsf{R}^{2}=0.9952$ for acid treated standards and $y=0.0007\\times+$ 8E-05, ${\\mathsf{R}}^{2}=0.9988$ for neutralized buffers) (Figure 1).  \n\n# Effect of acid treatment on alkaline phospha tase activity in serum  \n\nTo confirm if addition of dilute acid (0.2 N HCl) could affect ALP activity, human serum was used as the source of ALP enzyme. The ALP activity of serum represents the total ALP of liver, kidney and bone functions. Each serum sample was used in two sets, one with acid fol­ lowed by neutralization and another with neu­ tralized buffer to see the effect of acid on ALP activity. There was no significant difference in ALP values of serum collected from 8 human subjects, when acid was added as shown in Figure 2.  \n\n![](images/e3323f6b754846cf4da7bec63aaa1e79e2c27081dcd655d7706b1c062feff229.jpg)  \nFigure 2. Effect of acid treatment on ALP activity in serum samples. Serum samples were mixed with AMP buffer and incubated for 15 minutes and HCl was added followed by neutralization in one set and in other only neutralized buffer was added. Reaction was stopped and OD was taken at 405 nm. Total vol­ ume was $0.320\\mathrm{mL}$ . $^\\star p$ value- $<0.05$ .  \n\nComparison between DNA fragmentations by acid treatment vs sonication on ALP activity  \n\nSome methods [13] rely upon sonication for lysis of cells to estimate ALP activity and also for protein estimation. This method does not pose any viscosity problems. However, sonica­ tion itself is a very aggressive method which may affect the enzyme activity due to rapid vibration of protein molecules or by localized heating during sonication. To check whether sonication would affect the enzyme activity, ALP activity was determined from serum with sonication, in presence and absence of neutralization. ALP was estimated by sonication followed by modified protocol i.e. addition of acid followed by neutralization. Both the meth­ ods were compared with their respective con­ trol conditions viz., using neutralized buffer for acid followed by neutralization and non-sonica­ tion for sonicated samples. Addition of acid to the sample followed by neutralization affected ALP but not to that extent as compared to neu­ tralized buffer (Figure 3A). Studies have shown that sonication improves estimation of ALP and similar results were obtained when sonicated and non-sonicated samples were compared in Figure 3B, 3C.  \n\n![](images/870bcd9b3f1e2acc06200d0cd944a108e2d0b23b29a27b7c9e2ddd6b06e4a190.jpg)  \n\n![](images/fa7e373bdc0eb78f6a75ed8da491848578d686bca41adfb3556d5e9c7b9bb45e.jpg)  \nFigure 4. Compatibility between AMP buffer and Bradford assay. To the BSA standards $(0.5{-}3.0~\\upmu\\mathrm{g})$ , $250~\\upmu\\updownarrow{}$ of Bradford reagent added in presence and absence AMP buffer. Linearity was obtained up to $3.0\\upmu\\mathrm{g}$ . Total volume was $0.310\\mathrm{mL}$ .  \n\n# Compatibility of AMP buffer in Bradford assay  \n\nTotal protein concentration has to be estimated in sample to express the ALP activity/mg of protein for comparison. Bradford reagent changes its color to blue at alkaline pH. The compatibility between Bradford and AMP buffer with HCl and NaOH as ALP reaction is a mixture of these three components was checked. It was observed that there was no change in color when AMP buffer $\\l(y~=~0.0551{\\tt x}$ - 0.0192, $\\mathsf{R}^{2}=0.9776)$ was added to the Bradford reagent (Figure 4) as compared to without AMP buffer $(y=0.051x\\cdot0.0159$ , $\\mathsf{R}^{2}=0.9887;$ ) and to that somehow HCl has additive effect to be used in the modification step of the existing protocol. We got linearity from 0.5 to $3~{\\upmu\\mathrm{g}}$ of BSA with $250~\\upmu\\updownarrow$ of Bradford reagent.  \n\nOptimization of cellular alkaline phosphatase activity using HepG2 (hepatocellular carci­ noma, human) and human fetal osteoblastic (hFOB/ER9) cell lines  \n\nBone and Hepatic tissues are the source of alkaline phosphatase. Total ALP activity was expressed in U/mg of protein. HepG2 cells and hFOB/ER9 cells were cultured in a 96 well plate and ALP activity was estimated after 15 mins. The total ALP activity in HepG2 cells (673 U/mg of proteins) and hFOB/ER9 $(22.13~\\mathsf{U/m g}$ of protein) cells were estimated using proposed protocol (Figure 5). The main advantage being direct determination of ALP activity in 96 well microtiter plate in which the cells were cultured.  \n\n![](images/190ccd691f9d2be31065cf0f11be57826bba743e50729923575a0b268199b14e.jpg)  \nFigure 5. ALP activity in HepG2 and hFOB/ER9 cells. Cells lysed in AMP buffer with pNPP substrate and incubated for different time period. After incubation HCl was added followed by addition of NaOH to neu­ tralize and stopped the ALP reaction. Total volume was 0.210 mL.  \n\n# Discussion  \n\nHarris and group in 1995 [10] published the protocol of estimating ALP activity in osteoblastic cells directly using AMP buffer, which was modified later by same group [13] in which the cells were trypsinized and lysed by sonica­ tion to reduce the DNA viscosity. The ALP activ­ ity was expressed as ALP activity/DNA content in which DNA content was estimated by using a kit based Picogreen dye method. Although this method seems to work, there may be sources of errors in taking equal number of cells for trypsinization and differences in extent of soni­ cation while handling countless batches of cell cultures. To make the method less laborious, error free and cost efficient some new changes were included in our study. This includes addi­ tion of HCl to final concentration of $0.2\\mathrm{~N~}$ which brings about depurination of the DNA followed by NaOH addition, to break the DNA into small fragments, thereby reducing the DNA viscosity. Before implementation of these modifications to existing protocol, each and every condition of this assay was optimized critically.  \n\nFirst, addition of 0.2 N HCl followed by neutral­ ization with 0.2 N NaOH should not affect stan­ dards of assay i.e. para nitro phenol (PNP) and by comparing standards in presence of HCl and neutralized HCl, it was observed that addition of HCl did not affect the standard as shown in Figure 1. Neutralized buffer which includes a mixture of HCl and NaOH was considered as control for its experimental condition. Once it was clear that standards were not affected by HCl addition, this new approach was applied to estimate ALP activity in serum samples and it was found that the ALP activity was not affect­ ed significantly by addition of acid to the serum (Figure 2).  \n\nSecond, ALP activity was estimated in serum samples by two methods, one in which $0.2\\mathrm{~N~}$ HCl was added to one half of the serum and the other half samples were sonicated before ALP estimation. Although, sonication has improved ALP activity to some extent similar to previous study [18], ALP assay after sonication can be tedious and time consuming especially when numerous cell cultures have to be handled. Secondly, when many samples are being han­ dled one after another the extent of sonication may not be the same each time and this may lead to erroneous results. An important advan­ tage of the AMP buffer lysis method is that cells need to be cultured only in plates where they are directly lysed by addition of AMP lysis buf­ fer. On the other hand, for sonication, the cells have to be scrapped before collecting them into a tube and this scrapping step may not be the same in all the plates. This was also a source of error while assaying ALP activity. From this comparison it was observed that though sonication improved the ALP activity as compared to non-sonicated samples addition of HCl did not cause any significant change in the ALP activity as shown in Figure 3.  \n\nThird, it has been clearly shown in Figure 4 that even Bradford assay was not affected by addi­ tion of HCl containing AMP buffer. Fourth, this modified protocol gives advantage over existing protocol in absolute protein estimation as the difficulty of sample collection after ALP in case of cells was reduced by modification opted in the study. Lastly, ALP activity can be estimated in microtiter plate or microfuge tubes and can be constructed conveniently to fulfill many objectives at the same time.  \n\nALP activity is estimated to assess the function of hepatocytes, kidney cells and osteoblastic activity of osteoblasts [12, 19, 20]. It is also  \n\n# Detection of ALP in cellular model systems  \n\nused as diagnostic marker in liver and bone diseases [21, 22]. The adapted protocol should not cause any impediments while working with cell lines. To address this, the protocol was optimized with HepG2 and hFOB/ER9 cell lines. Hepatic carcinoma cells express high levels of ALP compared to fetal cells as shown in Figure 5. The results presented here clearly show that ALP activity was not affected by this newly mod­ ified and optimized protocol for ALP estimation from cell culture model systems. The advantag­ es of this method are less labor-intensive, cost effective, fast and more importantly estimation in single tube to minimize experimental errors.  \n\n# Limitation and conclusion  \n\nThe main limitation was the optimization of the protocol in only limited number of cell lines. Also, the enzyme kinetics (at different time peri­ od) was not performed due to limited availabil­ ity of funds and resources.  \n\nTo conclude, the modified protocol presented here is more efficient, economical, fast and consistent method to estimate ALP activity and protein concentration in a single tube for cul­ tured cells.  \n\n# Acknowledgements  \n\nWe are grateful to all the study subjects who participated in the present study and the labo­ ratory staff for their technical support. The fel­ lowship of Poonam Kanta was supported by CSIR, New Delhi (09/141/(0171)/2010-EMR1). This research did not receive any specific grant from funding agencies in the public, com­ mercial, or not-for-profit sectors.  \n\n# Disclosure of conflict of interest  \n\nNone.  \n\nAddress correspondence to: Dr. Thungapathra Muthukumarappa, Department of Biochemistry, Postgraduate Institute of Medical Education & Research (PGIMER), Chandigarh 160012, India. E-mail: thun­ gapathra@gmail.com  \n\n# References  \n\n[1] Coburn SP. Alkaline phosphatase (EC3.1.3.1) in serum is inhibited by physiological concen­ trations of inorganic phosphate. J Clin Endocri­ nol Metab 1998; 83: 3951-3957.  \n\n[2]\t Malhotra R, Grover V, Kapoor A and Kapur R. Alkaline phosphatase as a periodontal disease marker. Indian J Dent Res 2010; 21: 531-536.   \n[3] Rajagambeeram R, Abu Raghavan S, Ghosh S, Basu S, Ramasamy R and Murugaiyan SB. Di­ agnostic utility of heat stable alkaline phos­ phatase in hypertensive disorders of pregnan­ cy. J Clin Diagn Res 2014; 8: CC10-CC13.   \n[4] Doyon A, Fischer DC, Bayazit AK, Canpolat N, Duzova A, Sözeri B, Bacchetta J, Balat A, Büscher A, Candan C, Cakar N, Donmez O, Dusek J, Heckel M, Klaus G, Mir S, Özcelik G, Sever L, Shroff R, Vidal E, Wühl E, Gondan M, Melk A, Querfeld U, Haffner D and Schaefer F. Markers of bone metabolism are affected by renal function and growth hormone therapy in children with chronic kidney disease. PLoS One 2015; 10: e0113482.   \n[5] Sharma U, Pal D and Prasad R. Alkaline phos­ phatase: an overview. Indian J Clin Biochem 2014; 29: 269-278.   \n[6] Herz F and Nitowsky HM. Alkaline phospha­ tase activity of human cell cultures: kinetic and physical-chemical properties. Arch Biochem Biophys 1962; 96: 506-515.   \n[7]\t Morgenstern S, Kessler G, Auerbach J, Flor RV and Klein B. An automated p-nitrophenylphos­ phate serum alkaline phosphatase procedure for the AutoAnalyzer. Clin Chem 1965; 11: 876- 888.   \n[8] Nowrouzi A and Yazdanparast R. Alkaline phos­ phatase retained in HepG2 hepatocarcinoma cells vs alkaline phosphatase released to cul­ ture medium: difference of aberrant glycosyl­ ation. Biochem Biophys Res Commun 2005; 330: 400-409.   \n[9] Bowers GN Jr and McComb RB. Measurement of total alkaline phosphatase activity in human serum. Clin Chem 1975; 21: 1988-1995.   \n[10]\t Harris SA, Enger RJ, Riggs BL and Spelsberg TC. Development and characterization of a conditionally immortalized human fetal osteo­ blastic cell line. J Bone Miner Res 1995; 10: 178-186.   \n[11]\t Sabokbar A, Millett P, Myer B and Rushton N. A rapid, quantitative assay for measuring alka­ line phosphatase activity in osteoblastic cells in vitro. Bone Miner 1994; 27: 57-67.   \n[12]\t Robinson JA, Harris SA, Riggs BL and Spels­ berg TC. Estrogen regulation of human osteo­ blastic cell proliferation and differentiation. Endocrinology 1997; 138: 2919-2927.   \n[13]\t Yen ML, Chien CC, Chiu IM, Huang HI, Chen YC, Hu HI and Yen BL. Multilineage differentiation and characterization of the human fetal osteo­ blastic 1.19 cell line: a possible in vitro model of human mesenchymal progenitors. Stem Cells 2007; 25: 125-131.   \n[14]\t Holmquist G. The mechanism of C-banding: depurination and $\\upbeta$ -elimination. Chromosoma 1979; 72: 203-224.   \n[15]\t Sambrook J and Russell DW. Southern blot­ ting: capillary transfer of DNA to membranes. CSH Protoc 2006; 2006: pdb.prot4040.   \n[16]\t Hossain S, Rahman M, Fatima N, Haque M and Islam J. Leucas zeylanica (L.) R. Br. protects ethanol and hydrogen peroxide-induced oxida­ tive stress on hepatic tissue of rats. ICPJ [Internet]. 4 Aug, 2013 [cited 5 Nov, 2020]; 2: 148-151. Available from: https://www.bangla­ jol.info/index.php/ICPJ/article/view/16076.   \n[17]\t Bradford MM. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem 1976; 721: 248-254.   \n[18]\t Qin S, Hu C and Oenema O. Differentiating in­ tracellular from extracellular alkaline phospha­ tase activity in soil by sonication. PLoS One 2013; 8: e58691.   \n[19]\t Guimarães GN, Stipp RN, Rodrigues TL, de Souza AP, Line SR and Marques MR. Evaluation of the effects of transient or continuous PTH administration to odontoblast-like cells. Arch Oral Biol 2013; 58: 638-645.   \n[20]\t De Luna-Bertos E, Ramos-Torrecillas J, GarcíaMartínez O, Guildford A, Santin M and Ruiz C. Therapeutic doses of nonsteroidal anti-inflam­ matory drugs inhibit osteosarcoma MG-63 os­ teoblast-like cells maturation, viability, and biomineralization potential. ScientificWorldJournal 2013; 2013: 809891.   \n[21]\t Kim SH, Shin KH, Moon SH, Jang J, Kim HS, Suh JS and Yang WI. Reassessment of alkaline phosphatase as serum tumor marker with high specificity in osteosarcoma. Cancer Med 2017; 6: 1311-1322.   \n[22]\t Garnero P and Delmas PD. Assessment of the serum levels of bone alkaline phosphatase with a new immunoradiometric assay in pa­ tients with metabolic bone disease. J Clin Endocrinol Metab 1993; 77: 1046-1053.  \n\nDetection of ALP in cellular model systems  \n\n# Assay of ALP activity in cellular model systems  \n\nObjective: Determination of cellular ALP activity directly in 96 well microtiter plate.  \n\nPrinciple of the method: Addition of 2-amino 2-methyl 1-propanol (AMP) buffer to the cells releases cel­ lular content including genomic DNA, proteins and enzymes. Alkaline phosphatase in the cell lysate converts paranitrophenyl phosphate (PNPP) present in the AMP buffer to yellow colored paranitrophenol (PNP) at pH 10.5 which is read at 410 nm. At the end of incubation, just before stopping the enzyme activity with NaOH, HCl is added to $0.2\\mathrm{~N~}$ . The addition of HCl to the cell lysate causes depurination of DNA and further fragmentation of DNA is induced when NaOH is added. This results in complete loss of viscosity due to the released genomic DNA, making it easier to pipette out an aliquot of the reaction content - after taking the spectrophotometric reading of ALP assay - for direct protein estimation by Bradford assay. Finally the ALP activity in the cells is expressed as µmols of PNP produced/min/mg of protein. All standards are set up the same way as the tests for both ALP and protein estimations.  \n\n# Reagents:  \n\na) AMP (2-amino 2-methyl 1-propanol) buffer pH 10.5: Add 11.6 mL of AMP to $60\\mathsf{m L}$ of distilled water and adjust the pH to 10.5 with 6 N HCI. The final volume is made up to ${\\underline{{100}}}{\\mathrm{mL}}$ with distilled water. This can be stored at $4^{\\circ}\\mathtt{C}$ for 6 months.  \n\nb) PNPP (paranitrophenyl phosphate) substrate: The substrate solution $(2{\\mathrm{~mg/mL}}$ ) is made in AMP buf­ fer and prepared freshly before use.  \n\nc) HCl: 6 N  \n\nd) Neutralised HCl: To $6m L$ of 6 N HCl, 3.6 mL of 10 N NaOH is added  \n\ne) NaOH: 1 N & 10 N  \n\nf) Standard PNP (paranitrophenol) stock solution (10.8 mmol/L): To $8m L$ of 1 N NaOH, $15~\\mathsf{m g}$ of PNP is dissolved and made up to $\\boldsymbol{10}\\mathrm{ml}$ . To be stored in a brown glass bottle at ambient temperature.  \n\ng) Working PNP $(54\\upmu\\mathrm{mol/L})$ ): To $0.995\\mathrm{mL}$ of 1 N NaOH solution, 0.005 mL of the PNP stock solution is added. To be prepared freshly before use.  \n\nh) Bradford reagent: Dissolve 100 mg Coomassie Brilliant Blue G-250 in $50\\mathrm{~m}|95\\%$ ethanol and then add $100~\\mathrm{m}|~85\\%$ (w/v) phosphoric acid. Once the dye has completely dissolved, dilute to 1 liter with deionised water and filter through Whatman #1 paper. The Bradford reagent is also available commercially.  \n\ni) Stock BSA (bovine serum albumin) solution $({\\bf10\\mathrm{mg/mL}})$ in 0.15 M NaCl: To ensure homogenous solu­ tion, let it stand at ${\\sim}25^{\\circ}\\mathrm{C}$ for several hours, with intermittent gentle shaking, before using it to make the working standard.  \n\nj) Working BSA standard solution $(200\\upmu\\mathrm{g/mL})$ ): The stock BSA solution is diluted 50X in 0.15 M NaCl.  \n\nk) ALP assay reagent blank for setting up BSA standards: Mix 0.100 mL of PNPP substrate in AMP buf­ fer, $0.006\\mathsf{m L}$ of 6 N HCl and 0.104 mL of 1 N NaOH  \n\nl) Equipment and plastic ware: Microplate reader, 96 well plates.  \n\n# Detection of ALP in cellular model systems  \n\n# A. Schematic Presentation of ALP activity determination in cells.  \n\nApproximately $\\mathsf{1}\\times\\mathsf{10^{4}}$ cells (or as required) per well are plated in a 96 well plate   \n↓   \nCells to be maintained/treated as per requirement, in triplicates, along with appropriate controls   \n↓   \nTo the wells containing only cells (after removing media) $0.1\\mathrm{mL}$ of AMP buffer containing   \nPNPP substrate is added. Addition of AMP results in cellular lysis.   \n↓   \nIncubate at ${37}^{\\circ}{\\mathrm{C}}$ for 15 minutes   \n↓   \nAdd 0.006 mL of 6 N HCl and mix the contents by briefly swirling the plate.   \n↓   \nQuickly add 0.104 mL of 1.0 N NaOH which insures the termination of ALP activity   \nas well as neutralization of HCl   \n↓   \nMeasurement at 405 nm in a microplate reader  \n\nSetting up PNP standards: Standards are also set up in triplicates in microtiter plate and read at 405 nm.  \n\nSetting up ALP assay of test samples:   \n\n![](images/e7d5accb8c11ce25cd3c2e1a37cdf04b967545f2e7a5587c34b3bd6e8bfd0b36.jpg)  \n\nTotal volume of standards\\*\\*: 0.210 mL.  \n\n![](images/64cd585c3a15cecafce19b755676d217bf17e3145714bfcc4397911b23ce563e.jpg)  \nTotal volume of ALP assay mix of tests\\*\\*: 0.210 mL  \n\n\\*PNP quantities in standards should be optimized such that test results are within the range of standards  \n\n\\*\\*Total volume and reagent composition of PNP standards and tests should be same thus obviating the need for reagent blank  \n\nTest absorbance X Standard PNP amount ALP activity in test (μmol/min) (A) $=$ Standard absorbance X 15  \n\nFor expressing the specific activity of ALP of individual samples, a small aliquot of the ALP assay mix is assayed for protein content by Bradford’s method as given below.  \n\n# Detection of ALP in cellular model systems  \n\nB. Flow chart for protein estimation by Bradford assay  \n\nA small aliquot $(10{-}50~\\upmu\\mathrm{L})$ of ALP assay mix (cell lysate) is added to the wells of a fresh plate ↓ Bradford Reagent $(250~\\upmu\\mathsf{L})$ is added to the wells containing the ALP assay mix ↓ The blue color developed is measured at 595 nm in a microplate reader  \n\nSetting up BSA standards: In place of cell lysate same volume of ALP assay reagent blank is used  \n\n![](images/a105edad8672ddbfdec5b6b5b0161e8d2c812d7994faa545760fe35391d82a4e.jpg)  \n\nTotal volume of standards $=0.310\\mathsf{m L}^{\\star\\star}$  \n\nSetting up Bradford assay of tests:   \n\n![](images/5cc37cece0a858d404a94535e40066fea1b777d7bda1cba7b48de95c69db754f.jpg)  \n\nTotal volume of tests $=0.310\\mathsf{m L}^{\\star\\star}$  \n\n$\\star_{\\mathsf{B S A}}$ standard quantities and the test aliquot volumes used for protein estimation should be optimized such that test results are within the range of standards  \n\n\\*\\*Total volume and reagent composition of BSA standards and tests in the Bradford assay should be same  \n\nTest absorbance X Standard BSA amount $({\\upmu}{\\upxi})$ X 210 Protein quantity (mg) in total cellular lysate (B) $=$ Standard absorbance X 10 X 1000  \n\nFinally the ALP activity of the cells in the micro plate wells is presented after normalizing across the wells with respect to the protein content of the lysates  \n\nA [ALP activity (units) in test (μmol/min)] ALP activity Units/mg of protein in test $=$ B [Protein quantity (mg) in total ALP assay mix]  "}